src/components/Metrics/IstioMetrics.tsx

```diff
@@ -155,6 +156,8 @@ class IstioMetrics extends React.Component<IstioMetricsProps, MetricsState> {
           return `${this.props.grafanaInfo.url}${this.props.grafanaInfo.workloadDashboardPath}?${
             this.props.grafanaInfo.varNamespace
           }=${this.props.namespace}&${this.props.grafanaInfo.varWorkload}=${this.props.object}`;
+        case M.MetricsObjectTypes.APP:
+          return undefined;
```

Comment:
Yes, I unified {App,Workloads,Service}MetricsContainer containers into IstioMetricsContainer. But I realized that this is needed since AppMetricsContainer always omit the grafana URL to hide links to Grafana.